pkg/cmd/admin/router/router.go

```diff
@@ -293,6 +299,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					{Type: dapi.DeploymentTriggerOnConfigChange},
 				},
 				Template: dapi.DeploymentTemplate{
+					Strategy: dapi.DeploymentStrategy{
```

Comment:
@liggitt  your concern's fair. Maybe the better thing is to have the ```-10%``` be the default - in which case it makes the need for this change moot.  @ironcladlou  what do you think?

Hmm @pweil-  yeah making the default check and set fields individually would be probably better but that said it might not be needed (at least as of yet) if the defaults suffice for most cases.